### PR TITLE
[55] Add Page Navigation to your Pages- Frontend

### PR DESCRIPTION
**Pull request by Onyinye101**
## Task: `[55] Add Page Navigation to your Pages`
- `Added Page Navigation`

Screenshot 2020-10-05 at 11 37 06 AM
Screenshot 2020-10-05 at 11 37 22 AM

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done @Onyinye101 

You are expected to link your already created contact form page and product page together such that if I am on the product page, I can go to contact form page and vice versa. You are not a create a new page here again which I noticed that's what you are doing.

File: submissions/onyinyeChukwuka/HTML-Forms/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
```
Comment (ainobs) [inline submissions/onyinyeChukwuka/HTML-Forms/index.html:1]: you're missing the DOCTYPE declaration here

**Review by ainobs — CHANGES_REQUESTED** [review]

**Comment by Onyinye101** [comment]
I don't understand what you are asking me to do. Which of the folders do I
make the changes to since the index.html and contact forms are in two
separate folders? which of them do I submit a PR for?

File: submissions/onyinyeChukwuka/HTML-Forms/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/onyinyeChukwuka/HTML-Forms/index.html]: @Onyinye101 You don't need to create a new product page. You only need to link the already created product page to contactform page and vice versa

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/onyinyeChukwuka/HTML-Forms/index.html (hunk 2 above)
Comment (Onyinye101) [reply submissions/onyinyeChukwuka/HTML-Forms/index.html]: Thank you, I have corrected it now.

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 3]
```diff
@@ -16,6 +17,7 @@
                     <li> <a href="#"> WOMEN</a></li>
                     <li> <a href="#"> MEN  </a></li>
                     <li> <a href="#"> Hot Sales </a></li>
+                    <li> <a href="../html-forms/contactform.html" target="_blank"> Contact Us </a></li>
```
Comment (ToluAkin) [inline submissions/onyinyeChukwuka/HTML/index.html:20]: @Onyinye101 Are you sure this works? Your html form folder name is different from what is here

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Onyinye101** [comment]
Yes, it works. I tested it.

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**